07-Linking-Accounts/Auth0Sample/ProfileViewController.m

```diff
@@ -179,7 +179,7 @@ - (IBAction)unlinkAccount:(id)sender {
     HybridAuth *auth = [[HybridAuth alloc] init];
     A0SimpleKeychain* keychain = [[A0SimpleKeychain alloc] initWithService:@"Auth0"];
 
-    [auth unlinkUserAccountWithIdToken:[keychain stringForKey:@"id_token"] userId:self.userProfile.sub identity:identity  callback:^(NSError * _Nullable error, NSArray<NSDictionary<NSString *,id> *> * _Nullable payload) {
+    [auth unlinkUserAccountWithAccessToken:[keychain stringForKey:@"access_token"] userId:self.userProfile.sub identity:identity  callback:^(NSError * _Nullable error, NSArray<NSDictionary<NSString *,id> *> * _Nullable payload) {
```

Comment:
Same here. I suspect this should be `[auth unlinkUserAccount:[keychain...` and if it was done/worked before this PR it wouldn't have changed.